site/app/controllers/grading/ImagesController.php

```diff
@@ -48,4 +54,139 @@ public function viewImagesPage() {
         $students = $this->core->getQueries()->getAllUsers();
         $this->core->getOutput()->renderOutput(array('grading', 'Images'), 'listStudentImages', $students, $grader_sections, $instructor_permission);
     }
+
+    /**
+     * @Route("/{_semester}/{_course}/student_photos/upload")
+     */
+    public function ajaxUploadImagesFiles() {
+        if(!$this->core->getUser()->accessAdmin()) {
+            return $this->core->getOutput()->renderResultMessage("You have no permission to access this page", false);
+        }
+
+        if (empty($_POST)) {
+            $max_size = ini_get('post_max_size');
```

Comment:
Could this be kept outside and later used instead of hardcoding on line 124?

Reply:
No. They are different.

`post_max_size` specifies the maximum size of things that PHP can handle. If a POST sends things bigger than this size, then PHP will fail to receive them. Instead, the POST field will be empty.

PHP's max size is different from what we want for the max size for one single image file.

Reply:
https://github.com/Submitty/Submitty/issues/3626 may be related to this. I am not sure about the current status of that issue though.